glib/src/subclass/types.rs

```diff
@@ -86,6 +86,18 @@ pub unsafe trait InstanceStruct: Sized + 'static {
     }
 }
 
+/// Trait implemented by any type implementing `ObjectSubclassIs` to return the implementation, private Rust struct.
+pub trait ObjectSubclassIsExt: ObjectSubclassIs {
+    fn impl_(&self) -> &Self::Subclass;
+}
+
+impl<T: ObjectSubclassIs<Subclass = S>, S: ObjectSubclass<Type = Self>> ObjectSubclassIsExt for T {
+    /// Returns the implementation (the private Rust struct) of this class instance
```

Comment:
You have to move this to the trait definition, not the impl block

Reply:
```suggestion
    /// Returns the implementation (the implementation, private Rust struct) of this class instance.
```

Reply:
but why repeat "the implementation"?

Reply:
Because I failed reading, sorry :)